src/platform/mouse.c

```diff
@@ -17,16 +18,18 @@ void system_mouse_get_relative_state(int *x, int *y)
 void system_mouse_set_relative_mode(int enabled)
 {
     if (enabled) {
+        if (data.enabled) {
```

Comment:
Suggestion: do the following check at the start of the function, so you don't need to have another check both here and in the `else if` below:
```
if (enabled == data.enabled) {
    return;
}
```